packages/ag-charts-community/src/scene/matrix.ts

```diff
@@ -1,5 +1,7 @@
 import { BBox } from './bbox';
 
+export const IDENTITY_MATRIX_ELEMENTS = Object.freeze([1, 0, 0, 1, 0, 0]);
```

Comment:
There's a performance cost to object.freeze - especially on arrays. You can mark this `as readonly number[]`